/**:
  ros__parameters:
    max_accel: -2.8
    max_jerk: -5.0
    system_delay: 0.5
    delay_response_time: 0.5
    is_publish_debug_path: false # publish all debug path with lane id in each module
